tools/system_libs.py

```diff
@@ -24,7 +24,8 @@ def run_commands(commands):
       call_process(command)
   else:
     pool = multiprocessing.Pool(processes=cores)
-    pool.map(call_process, commands, chunksize=1)
+    # https://bugs.python.org/issue8296
+    pool.map_async(call_process, commands, chunksize=1).get(maxint)
```

Comment:
i don't see this `maxint` hack explained either in this link or in the link in the description of the issue. did i miss it?